src/domain/QuizResult.ts

```diff
@@ -0,0 +1,18 @@
+export interface QuizResult {
+  currentLevel: {
```

Comment:
```suggestion
  currentLevel: Level
```

Why not use Level interface, since it has the same properties?